src/resources/MachineLearning/CCConfiguration/CCConfiguration.ts

```diff
@@ -0,0 +1,24 @@
+import API from '../../../APICore';
+import {New} from '../../BaseInterfaces';
+import Resource from '../../Resource';
+import {CCConfigurationModel} from './CCConfigurationInterfaces';
+
+export default class CCConfiguration extends Resource {
```

Comment:
Not sure about the `CCC` naming, I would go with the full name for the sake of clarity if `CCC` isn't going to be commonly used in the documentation.